edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/ConnectivityAwareClient.cs

```diff
@@ -116,23 +134,24 @@ async Task<T> InvokeFunc<T>(Func<Task<T>> func, string operation, bool useForCon
                     this.deviceConnectivityManager.CallSucceeded();
                 }
 
-                Events.OperationSucceeded(operation);
+                this.HandleDeviceConnectedEvent(this, EventArgs.Empty);
```

Comment:
This is a new operation?

Reply:
Yes, added that here to track when the device gets connected.